commerce_coordinator/apps/core/clients.py

```diff
@@ -66,36 +66,53 @@ def normal_timeout(self):
 
     def log_request_response(self, ext_logger, response):
         """
-        Log requests.Request Response.
+        Log requests.Request responses with detailed information.
 
         Args:
             ext_logger: The logger of the module the Client (or Client-
                 derived class) is running in.
             response: A successful Response object from the requests library.
         """
-        ext_logger.debug('Request URL: %s', response.request.url)
-        ext_logger.debug('Request method: %s', response.request.method)
-        ext_logger.debug('Request body: %s', response.request.body)
-        ext_logger.debug('Request headers: %s', response.request.headers)
-        ext_logger.debug('Response status: %s %s', response.status_code, response.reason)
-        ext_logger.debug('Response text: %s', response.text)
+        # Extract request details and response information
```

Comment:
Please remove these comments where added. We don't add this much comments in the codebase